src/components/plots/MultiViewGrid.jsx

```diff
@@ -1,39 +1,137 @@
-import React, { useEffect, useState, useRef } from 'react';
+import React, {
+  useEffect, useState, useRef, useCallback,
+} from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
 
 import _ from 'lodash';
 import { Col, Row, Space } from 'antd';
 import { getPlotConfigs, getCellSets } from 'redux/selectors';
 import { loadCellSets } from 'redux/actions/cellSets';
+import {
+  updatePlotConfig,
+  savePlotConfig,
+} from 'redux/actions/componentConfig/index';
 
 import PlatformError from 'components/PlatformError';
+import loadConditionalComponentConfig from 'redux/actions/componentConfig/loadConditionalComponentConfig';
+import Loader from 'components/Loader';
+import { loadPaginatedGeneProperties, loadGeneExpression } from 'redux/actions/genes';
+import { plotTypes, plotUuids } from 'utils/constants';
+
+const PROPERTIES = ['dispersions'];
```

Comment:
why `PROPERTIES` is a list, where you always use `PROPERTIES[0]`? Can you simply have it as a string?

Reply:
It can get multiple properties I suppose, thats how it has been since the beginning

Reply:
if we can have multiple items under`PROPERTIES`, then let's keep it a list